Cargo.lock

```diff
@@ -2720,6 +2721,16 @@ version = "0.2.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "68354c5c6bd36d73ff3feceb05efa59b6acb7626617f4962be322a825e61f79a"
 
+[[package]]
+name = "miniz-sys"
```

Comment:
Why this is added?

Reply:
It's depended by the [flate2](https://crates.io/crates/flate2) package.

Reply:
flate2 has been a part of dependency before but miniz-sys is added in this PR.

Reply:
Thanks for the reminding. I change flate2 to use zlib instead of miniz instead.